faenet/model.py

```diff
@@ -242,15 +177,11 @@ def __init__(
                 hidden_channels if "updown" not in self.mp_type else num_filters
             )
 
-        if self.mp_type == "simple":
-            self.lin_geom = nn.Linear(num_filters, hidden_channels)
-            self.lin_h = nn.Linear(hidden_channels, hidden_channels)
-
-        elif self.mp_type == "sfarinet":
+        if self.mp_type == "sfarinet":
```

Comment:
I wouldn't keep references to sfarinet, it should either disappear or be renamed IMHO. What do you think @AlexDuvalinho ?

Reply:
Okok